src/test/java/io/confluent/connect/jdbc/dialect/BaseDialectTest.java

```diff
@@ -155,6 +158,18 @@ protected JdbcSourceConnectorConfig sourceConfigWithUrl(
     connProps.put(JdbcSourceConnectorConfig.TOPIC_PREFIX_CONFIG, "test-");
     connProps.putAll(propertiesFromPairs(propertyPairs));
     connProps.put(JdbcSourceConnectorConfig.CONNECTION_URL_CONFIG, url);
+    if (quoteTableNames != null) {
```

Comment:
No, because the tests change `BaseDialectTest.quoteTableNames` and then call this method. That's how test methods can test different kinds of quoting behavior.